src/screens/Login/form.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <div className="form">
```

Comment:
@noor-ahmed Should we use `form` here instead of `div`?  🤔
I think it'd be nice if we use `form`'s behavior.

Reply:
updated 🙌